getting_started/step_by_step/7_room_two.rst

```diff
@@ -207,5 +234,35 @@ Test the scene. You should now be able to click on the door to leave the pub -
 you will appear in the park. If you click on the left hand side of the park
 you will appear in the doorway of the pub.
 
-Additionally, due to the light map, you should be in shadows when you walk
-in the park, and get lighter when you walk under the light.
+Additionally, due to the light map configured for the park scene, you should be
+in shadows when you walk in the park, and get lighter when you walk under the
+light.
+
+Creating an exit using an ESCItem instead of an ESCExit
+-------------------------------------------------------
+As mentioned earlier, for more complex exit sequences, an ``ESCItem`` is used
+rather than an ``ESCExit``.
+
+If you wanted to create the pub door using an ``ESCItem`` you would:
+* Create an ``ESCItem`` as a child of the ``ESCRoom``. Give it a Global ID of
+"pub_door_item".
+* Create a new ESC script file (a text file with an ".esc" extension)
+and call it "pub_exit.esc". Where you place this file is up to you and how
```

Comment:
I think, we should move this information about organizing ESC scripts (you can place the ESC scripts where you want) into the step by step chapter "The first run" as ESC is described there.